composer-check-dependencies.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+DIRECTORY=$1
+
+if [ "$#" -eq 0 ]
+then
+ echo "Missing directory argument"
+ exit 1
+fi
+
+CHECK_RESULT_CODE_TEXT=`vendor/bin/composer-require-checker check $DIRECTORY/composer.json --ignore-parse-errors`
+CHECK_RESULT_CODE=$?
+if [ "$CHECK_RESULT_CODE" -eq "0" ]
+then
+ exit 0
+fi
+
+SYMBOLS_FOUND=`echo "$CHECK_RESULT_CODE_TEXT" | tail -n '+6' | head -n '-1' | sed 's/| //' | sed 's/ .*$//' | sort`
+
+IGNORE_FILE=$DIRECTORY/composer-check-dependencies-ignore.txt
+
+if [ ! -f $IGNORE_FILE ]
```

Comment:
Isn't this a DRY violation? I would get `$SYMBOLS_NOT_IGNORED` even if there is no ignore file and continue from there.

Reply:
fixed.. maybe ;-)